configure.ac

```diff
@@ -529,6 +529,8 @@ AH_VERBATIM([FLAC_API_EXPORTS],
 #else
 # define FLAC_API_EXPORTS
 #endif
+#elif defined(_WIN32)
+# define FLAC__NO_DLL
```

Comment:
Note to self: I vaguely remember this didn't work properly in some configuration because FLAC__NO_DLL was already set. Need to check this.